### ISSUE TITLE
Retrieve nodes positions

### ISSUE THREAD

**Issue by fpom** [cytoscape/ipycytoscape#333]
How can I retrieve from Python the positions of the nodes in a graph?

When a layout is applied, the field `position` in `.graph.nodes` objects is not updated. So far, the only way I've found is to bind a `mouseover` event on nodes and get the positions from the events, but this is not convenient as it requires specific actions from the user.

**Comment by TermeHansen** [comment]
I would really like to option to save a layout of the graph after one has re-arranged it to a specific liking.

Is this really not possible?

Even if we get the positions updated from python side, I guess next problem is to set them https://github.com/cytoscape/ipycytoscape/issues/254

**Comment by fpom** [comment]
I answered #254: I have no problem moving nodes from Python, it requires assigning them positions and then calling the `relayout` method.

**Comment by TermeHansen** [comment]
Nice, but still no way to get node positions without mouse over user interaction?

**Comment by fpom** [comment]
Indeed, I'm still unable to get the nodes' positions, only to set them. So frustrating...
